Interactive_Metrics.ipynb

```diff
@@ -274,20 +209,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def explore_peit_var(ms_var, occ_var, FE_var):\n",
-    "    cdf_plot([results['peit1']['combined'],calculate_peit_var(ms_var, occ_var, FE_var)],['Baseline','Modified'],\"PEIT1 [L/km]\",['gold','red'])\n",
+    "def get_modified_pphpl(**kwargs):\n",
+    "    # let's say that the original mean was 10, new mean is 20. we want to add 10 to all values\n",
+    "    change_mean = lambda param_mode: change_mean_from_kwargs(param_mode, kwargs)\n",
     "    \n",
-    "interactive_plot = interactive(explore_peit_var, ms_var=(-3.0, 3.0,0.5), occ_var=(-3, 3, 0.5), FE_var=(-3,3,0.5))\n",
-    "output = interactive_plot.children[-1]\n",
-    "output.layout.height = '640px'\n",
-    "interactive_plot"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Interaction"
+    "    pvt_pphpl_mod = pphpl_by_mode(change_mean(\"vphpl_pvt\"), params['mpw']['pvt'], change_mean(\"occ_pvt\"))\n",
+    "    bus_pphpl_mod= pphpl_by_mode(change_mean(\"vphpl_bus\"), params['mpw']['bus'], change_mean(\"occ_bus\"))\n",
+    "    wb_pphpl_mod=pphpl_by_mode(change_mean(\"vphpl_wb\"), params['mpw']['wb'], 1)\n",
```

Comment:
why are the mpw params not configurable here? Please add an explanation to the notebook as a markdown cell or as a python comment,